docker/scheduler

```diff
@@ -13,13 +13,14 @@ LABEL maintainer="wesley.monte@ccc.ufcg.edu.br"
 ENV MAIN_CLASS="org.fogbowcloud.saps.engine.core.scheduler.SchedulerMain"
```

Comment:
why this file name is not Dockerfile? We should standardize it in one of the possible approaches

Reply:
It was done in the #4.